pkcs11.opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "hex" { >= "1.0.0" }
   "integers"
   "ppx_deriving" { >= "4.0" }
-  "ppx_deriving_yojson" { >= "3.0" }
+  "ppx_deriving_yojson" { >= "3.5.1" }
```

Comment:
Is 3.5.1 really needed?  My impression is that 3.4 is also compatible.  Usually, declared dependencies of libraries are as wide as possible to avoid conflicts with declared dependencies of other libraries.

Reply:
Good spot, downgraded